pages/en/lb2/Deploy-for-GDPR-readiness.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Deploy for GDPR Readiness"
+layout: page
+keywords: LoopBack, GDPR
+tags: gdpr
+lang: en
+sidebar: lb2_sidebar
+permalink: /doc/en/lb2/Deploy-for-GDPR-readiness.html
+summary: LoopBack considerations for GDPR readiness.
+---
+
+# LoopBack considerations for GDPR readiness
+ 
+___________________________________________________________________________________________________
+### For PID(s): UT:30AH1
+
+## Notice:
+
+This document is intended to help you in your preparations for GDPR readiness. It provides information about features of LoopBack that you can configure, and aspects of the product’s use, that you should consider to help your organization with GDPR readiness. This information is not an exhaustive list, due to the many ways that clients can choose and configure features, and the large variety of ways that the product can be used in itself and with third-party applications and systems.
+
+__Clients are responsible for ensuring their own compliance with various laws
+and regulations, including the European Union General Data Protection Regulation. 
+Clients are solely responsible for obtaining advice of competent legal counsel as to 
+the identification and interpretation of any relevant laws and regulations that may 
+affect the clients’ business and any actions the clients may need to take to comply 
+with such laws and regulations.__
+
+__The products, services, and other capabilities
+described herein are not suitable for all client situations and may have restricted 
+availability. IBM does not provide legal, accounting, or auditing advice or represent or 
+warrant that its services or products will ensure that clients are in compliance with 
+any law or regulation.__
+
+
+___________________________________________________________________________________________________
+
+### Table of Contents
+1. GDPR
+2. Product Configuration for GDPR
+3. Data Life Cycle
+4. Data Collection
+5. Data Storage
+6. Data Access
+7. Data Processing
+8. Data Deletion
+9. Data Monitoring
+10. Responding to Data Subject Rights
+
+
+
+___________________________________________________________________________________________________
+### GDPR
+
+General Data Protection Regulation (GDPR) has been adopted by the European Union (“EU”) and applies from May 25, 2018.
+
+#### Why is GDPR important?
+
+GDPR establishes a stronger data protection regulatory framework for processing of personal data of individuals.  GDPR brings:
+
+* New and enhanced rights for individuals
```

Comment:
consistent bullets: `-`

Reply:
will ignore because it's part of the template